### DISCUSSION TITLE
Query custom field type in graphql

### DISCUSSION THREAD

**Discussion by zerinoid** [wp-graphql/wpgraphql-acf#222] [Q&A]
I'm trying to query a custom Field Type but in graphql settings it says:

![image](https://github.com/wp-graphql/wpgraphql-acf/assets/37341660/4cfd3421-80ac-4c79-9b78-e7fcb46a5ace)

But link wont take me to any guide, its broken. 

Is there truly a guide for this?

**Comment by alancwoo** [comment]
Also struggling with this and the link in the CMS doesn't point anywhere.

However, the readme mentions using the [`register_graphql_acf_field_type` API](https://github.com/wp-graphql/wpgraphql-acf#i-have-an-acf-extension-that-adds-a-new-field-type-will-it-work-with-wpgraphql-for-acf).

If you take a look at [how some of the other ACF fields are registered](https://github.com/search?q=repo%3Awp-graphql%2Fwpgraphql-acf%20register_field_type&type=code), it helps, for example the ACF select field:

```
<?php
namespace WPGraphQL\Acf\FieldType;

class Select {

	/**
	 * Register support for the "select" ACF field type
	 */
	public static function register_field_type(): void {
		register_graphql_acf_field_type(
			'select',
			[
				'graphql_type' => [ 'list_of' => 'String' ],
			]
		);
	}
}
```

So in the case of my custom select field, I have:

```
<?php
add_action("wpgraphql/acf/registry_init", function () {
    register_graphql_acf_field_type("example_custom_field", [
        "graphql_type" => ["list_of" => "String"],
    ]);
});
```

Also some [further notes in the docs](https://acf.wpgraphql.com/adding-support-for-3rd-party-acf-field-type/), though they are in the process of being updated it says.

Nonetheless, this link should be corrected in the plugin itself so it points somewhere useful.
